styles/_grid.scss

```diff
@@ -0,0 +1,8 @@
+.grid-item {
+  flex-grow: 1;
+  min-width: 20rem;
+  max-width: 28%;
```

Comment:
This may be throwing off your gridding. You have a max and min width set, but no desired regular `width` rule set.

This means the grid item is only defined by its limits!